azure-pipelines.yml

```diff
@@ -31,26 +31,42 @@ steps:
   displayName: 'Build image'
   env:
     DOCKER_OVERRIDE: $(dockerOverride)
+    dockerHubUsername: $(dockerHubUsername)
 
 - script: |
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake spec SPEC_OPTS="--format RspecJunitFormatter --out ./rspec-results.xml"'
   displayName: 'Execute tests'
   env:
     DOCKER_OVERRIDE: $(dockerOverride)
+    dockerHubUsername: $(dockerHubUsername)
 
 - script: |
-    $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec govuk-lint-ruby app config db lib spec --format clang"
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec rake lint_ruby"
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec rake lint_scss"
   displayName: 'Execute linters'
   env:
     DOCKER_OVERRIDE: $(dockerOverride)
+    dockerHubUsername: $(dockerHubUsername)
```

Comment:
Why do you need this variable?

Reply:
Ensures the env var is available for the docker-compose.az.yml file